pkg/testutil/test_suite_runner.go

```diff
@@ -116,15 +116,19 @@ func (runner *TestSuiteRunner) runTestScenario(t *testing.T, scenarioName string
 	runner.assertExpectations(assert, expectation, actual, err)
 }
 
-/* assertExpectations validates the actual outcome against the expected outcome.
+/*
+	assertExpectations validates the actual outcome against the expected outcome.
```

Comment:
nit: any reason for this change? Looks off when viewing the file.

Reply:
some auto-formatting issue when I save using visual-studio, will edit

Reply:
does this need to be reverted?